client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -2,7 +2,6 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
```

Comment:
I'm guessing that this is still a WIP, but `localize` applies here as it is necessary for the translatable strings within this file. 

<img width="596" alt="Screenshot 2020-05-06 at 09 25 11" src="https://user-images.githubusercontent.com/43215253/81154272-a7f7b680-8f7b-11ea-879f-7064e32006ba.png">

Reply:
Thank you very much for answering!
You're right, I am a newbie =P